src/gui/transferlistwidget.h

```diff
@@ -123,6 +123,7 @@ protected slots:
     void confirmRemoveAllTagsForSelection();
     QStringList askTagsForSelection(const QString &dialogTitle);
     void applyToSelectedTorrents(const std::function<void (BitTorrent::TorrentHandle *const)> &fn);
+    bool hasTorrentPreviewableFiles(const BitTorrent::TorrentHandle *const torrent) const;
```

Comment:
The function name makes no sense.

Reply:
What do you propose?

Reply:
The obvious correction would be to make it "torrentHasPreviewableFiles"; although "has" is incorrect verb choice here,  "torrentContainsPreviewableFiles" seems to be a bit too long.

Reply:
@zeule, please read [this](https://github.com/qbittorrent/qBittorrent/pull/9756#discussion_r228714891) thread. This function shouldn't be a member of TransferListWidget class.

Reply:
has and is should be everytime at beginning for functions which return bool

Reply:
isTorrentContainPreviewableFiles

Reply:
"does"

Reply:
isTorrentPreviewable() :)

Reply:
Oops... my bad English.

But I think using is/has prefer is preferred way but it isn't required if it is inappropriate. Otherwise we will produce really "monstrous" names.

IMO, torrentContainsPreviewableFiles is acceptable.

Reply:
>isTorrentPreviewable() :)

if we assume that "torrent" is used in a figurative sense, it will do.

Reply:
Renamed to torrentContainsPreviewableFiles